bundles/org.openhab.binding.tplinkrouter/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tplinkrouter"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="TD-W9970">
```

Comment:
I am asking myself if you should not use a more generic thing type ID ?
As you mentioned in the README, it could work for other routers than this particular one.
I would suggest "router".